platform/administer/templates/overview.mdx

```diff
@@ -0,0 +1,55 @@
+
+Because templates can be versioned, they support upgrade workflows. When a template is updated, those changes can be propagated to any virtual clusters based on it. This allows teams to update cluster configurations and installed apps consistently across environments.
+
+
+## Template types
+
+vCluster Platform supports multiple types of templates. Each one serves a different purpose in the cluster lifecycle:
+
+| Template type                | Custom resource                                           | Purpose                                                                                                                   | Common Use Cases                                                                         |
+| ---------------------------- | --------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| Virtual cluster              | `VirtualClusterTemplate`                                  | Specifies how virtual clusters are provisioned, including control plane settings, synchronization behavior, and metadata. | Provision virtual clusters with consistent architecture, features, and default settings. |
+| App                          | `AppTemplate`                                             | Defines reusable application configurations, such as Helm charts or container specifications.                             | Deploy standard applications across projects or spaces using consistent settings.        |
+| Namespace                    | `SpaceTemplate`                                           | Predefines configurations for new namespaces, including role-based access control, resource quotas, and labels.           | Automatically apply baseline policies when a namespace is created.                       |
+
+
+You can [create templates](/platform/next/administer/templates/create-templates) using the vCluster Platform UI for guided configuration or by applying manifests directly with kubectl, Helm, or ArgoCD. 
+
+
+## Apply parameters and versioning
+
+Templates support optional parameterization and versioning. Optionally, you can apply 
+
+- [Parameters](/platform/next/administer/templates/parameters): Enable dynamic customization of template fields, allowing users to specify values for CPU limits, labels, environment types, and other configuration options during vCluster provisioning.
+
+- [Versioning](/platform/next/administer/templates/versioning): Enables you to manage and iterate template configurations across different releases, providing version control for your cluster templates.
```

Comment:
Versioning also allows for optional automatic update of virtual clusters based on those versioned templates. Non-versioned templates do not support automatic update of virtual clusters.